Formula/cayley.rb

```diff
@@ -11,46 +11,58 @@ class Cayley < Formula
     sha256 "4177fdcb60422d484f1377e5367844ebcc9be471fffc76e717d8ad90e49ee99c" => :high_sierra
   end
 
-  depends_on "bazaar" => :build
   depends_on "go" => :build
-  depends_on "mercurial" => :build
+  depends_on "gobuffalo/tap/packr" => :build
+
+  CONFIGURATION_FILE = "cayley.json"
```

Comment:
Please just type these manually everywhere. This makes the formula different from other formula which hinders maintainability.